### ISSUE TITLE
Returns to First Slide After "submit" in Slidy

### ISSUE THREAD

**Issue by lecy** [rstudio/learnr#217]
I am trying to embed questions inside of a slidy presentation. 

Every time you answer a question, though, it sends you back to the first slide. Thoughts? 

[WomenMenQuestions_V3_slidy.zip](https://github.com/rstudio/learnr/files/2959455/WomenMenQuestions_V3_slidy.zip)

**Comment by lecy** [comment]
The submit button is rendered as follows in the HTML file:

```
<a href="#" class="button checkAnswer btn btn-primary btn-md" role="button" style="display: none;">Submit Answer</a>
```

**Comment by schloerke** [comment]
I'm removing slidly from learnr in the next release.  This should be resolved in #194.

**Comment by lecy** [comment]
Thank you. And thanks for a great package!
